nextjs/pages/components/ToggleButton.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 
-type props = {
-    isToggled: boolean,
-    setisToggled: React.Dispatch<React.SetStateAction<boolean>>, // react useState setter
-    label: string,
+type Props = {
+    isToggled: boolean;
+    setIsToggled: React.Dispatch<React.SetStateAction<boolean>>; // react useState setter
+    label: string;
```

Comment:
Til en type skal det være commaer
```suggestion
type Props = {
    isToggled: boolean,
    setIsToggled: React.Dispatch<React.SetStateAction<boolean>>, // react useState setter
    label: string,
```

Reply:
nop

Reply:
jo